scripts/create.js

```diff
@@ -8,7 +8,8 @@ const claspJSONPath = path.join(rootDir, '.clasp.json')
 // run clasp create if have not .clasp.json
 if (!fs.existsSync(claspJSONPath)) {
   const clasp = require.resolve('.bin/clasp')
-  spawn.sync(clasp, ['create'].concat(process.argv.slice(2)), {
+  const [, , title, parentId] = process.argv
```

Comment:
`[, , title, parentId]` より `slice(2)` の方が意図が見える気がしました。

```suggestion
  const [title, parentId] = process.argv.slice(2)
```